src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1028,9 +1029,22 @@
                     Condition="'%(PackageFile.IsSourceCodeFile)'!='true'" />
     </ItemGroup>
   </Target>
+  
+  <Target Name="GetPackageReport"
```

Comment:
It's right here: https://github.com/dotnet/buildtools/pull/1126/files#diff-03c502c8604ae1b4ae2efeb1f3d0c45eR1047
Also, now here: https://github.com/dotnet/buildtools/pull/1126/files#diff-03c502c8604ae1b4ae2efeb1f3d0c45eR131